### ISSUE TITLE
Outlook addin page.  Adding custom Dropdown element make page looks scrambled

### ISSUE THREAD

**Issue by LookerClocker** [OfficeDev/office-js#2665]
**When adding the custom dropdown element into addin page. It looks a bit weird only on Native Outlook with specific version.**
_**Version: 2016 (16.0.5197.1000) MSO (16.0.5278.1000)**_

## Your Environment

It doesn't work for Windows Native Outlook.  **But on Web Outlook (all versions) its working fine** 

**Microsoft Outlook Native App
Version: 2016 (16.0.5197.1000)**
![2022-04-03_15h06_08](https://user-images.githubusercontent.com/4919709/175945932-1b357994-fa81-4e3b-a956-2080dc717905.png)

 
## Expected behavior
Should render dropdown element.
<img width="393" alt="Screenshot 2022-06-27 at 15 42 44" src="https://user-images.githubusercontent.com/4919709/175945140-70726cae-bbfe-4657-81ec-b57aeb2b4c95.png">

## Current behavior
![2022-06-01_10h48_44](https://user-images.githubusercontent.com/4919709/175945042-44ec8d32-8b64-44a8-8a56-b080b86a924c.png)

## Context
I have custom Outlook addin. And Im trying to display custom dropdown there.

Im using React.js with styled components
```
"react": "^16.8.4",
"@xstyled/styled-components": "^1.16.0",
```

But Instead of seeing just dropdown element with some options, I see some black border with system window.. 

So basically it works well on all platforms and devices but only doesn't work on specific version of Outlook Native Application, **_with version 2016 (16.0.5197.1000)_**. Also the all details is in screenshot attachment.

Thanks!

## Useful logs

- [ ] Console errors
- [x] Screenshots
- [ ] Test file (if only happens on a particular file)

Thank you for taking the time to report an issue. Our triage team will respond to you in less than 72 hours. Normally, response time is <10 hours Monday through Friday. We do not triage on weekends.

**Comment by preethikakiru** [comment]
Hi @LookerClocker, thank you for making us aware of this issue! I was wondering if it would be possible to see clearer images of the expected output vs the custom dropdown element's current behavior? 
Best, 
Preethika

**Comment by LookerClocker** [comment]
Hi @preethikakiru ! Thanks for reply! Sending to you unblured screenshots:
This is how it looks now on **Outlook Native Application, with version 2016 (16.0.5197.1000)**
![2022-04-03_15h05_02](https://user-images.githubusercontent.com/4919709/176122319-da052efe-a1e6-415d-a1b8-e3c458f8eedb.png)
so, as you can see there is a dropdown but also we got some black border of some window


**But we're expecting to see this:** 

<img width="435" alt="Screenshot 2022-06-28 at 10 44 19" src="https://user-images.githubusercontent.com/4919709/176122896-656550ed-bf05-45e6-b787-07387e0d95c2.png">

Proper screen with closed dropdown without any weird border. 
Thank you!

**Comment by preethikakiru** [comment]
Ok great thanks! I have assigned this issue to @JuaneloJuanelo, who will be able to give you more information and help you get to a resolution.

**Comment by eran-levy** [comment]
Hi @preethikakiru @JuaneloJuanelo I wonder if there is any way to assist us in this?

**Comment by LookerClocker** [comment]
Hey @preethikakiru @JuaneloJuanelo did you have a chance to take a look on it? 
Thanks!

**Comment by Oleg-O** [comment]
Hi @LookerClocker, apologies for the delay in following up here. Could you please provide some clarifications:

1. What **Windows OS version** does this reproduce on? Do you have Microsoft Edge installed on this PC?
2. Do you observe this on more recent versions on Windows Desktop Outlook for your add-in?
3. Do you observe this with **any other add-in** - you could try a few free add-ins from store?
4. Do you observe this only when you add your drop down in your add-in UI? Just to confirm, are you saying this problem does not reproduce if you remove the drop down from your UI?
5. Does it reproduce consistently or sometimes?
6. Does it reproduce on a single PC or multiple PCs?
7. Does it reproduce for one user or multiple users?
8. Does resizing the add-in pane make the rendering issue disappear?
9. Does the PC that reproduces the issue use any OS scaling (there are OS settings that allow to scale the UI up for PCs with smaller but higher resolution monitors, so it's easier to read).

Is it possible to include full screenshot showing the issue (feel free to edit out content you don't want to share, but it'd be good to see the add-in pane and other panes of Outlook).

Thanks!

**Comment by LookerClocker** [comment]
Thanks @Oleg-O! Im on it!

**Comment by LookerClocker** [comment]
Hi, @Oleg-O 

Here the list of provided answers:

1. Windows 10
2. Seems like Yes
3. No
4. Only when Im adding my custom dropdown
5. Consistently
6. Single (since we have that specific version of Outlook only on single machine, so its depend of version of Outlook I believe)
7. One
8. No
9. No

Many thanks and sorry for delay!

**Comment by Oleg-O** [comment]
I wonder whether it'd be possible for you to create and share a simple ScriptLab snippet with your drop down to see if we could reproduce the issue? ScriptLab is a free Outlook add-in you can find in [Outlook add-in store](https://support.microsoft.com/en-us/office/get-an-office-add-in-for-outlook-1ee261f9-49bf-4ba6-b3e2-2ba7bcab64c8). It's a development tool were you can play with APIs and create and share snippets to help reproduce issues.

Thanks for your answers, it helps. For your answer on question #2 - you are saying that it does not reproduce just on Outlook 2016, but also more recent versions. Which versions have you tried and seen it reproduce on?

**Comment by LookerClocker** [comment]
Thansk, @Oleg-O  I'll try to run it with snippet and will let you know
